spec/src/modules/catalog/catalog-files.js

```diff
@@ -210,7 +210,6 @@ describe('ConstructorIO - Catalog', () => {
           variations: variationsBuffer,
           item_groups: itemGroupsBuffer,
           section: 'Products',
-          force: true,
```

Comment:
I added this previously in a separate PR and forgot to remove it.

Reply:
I saw this on the other PR but thought you left it there intentionally to reset the catalog.